raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
ect, examine and copy licenses, registers, records and any other document relating to this Law of any other law relating to the environment and the management of natural resources; [c] take samples in the form and manner prescribed by the Agency, or any articles and materials to which this Law relates and submit them for tests and analysis; [d] carry out periodic inspections of all establishments and undertakings within such District limits of jurisdiction which manufacture, produce as by-products, import, export, store, sell, distribute, or use any substances and materials that are likely to have significant impact on the environment to ensure that the provisions of this Law are complied with; [e] make examinations and inquiries so as to discover whether the provisions of this Law are being complied with; [f] carry out such other inspections as may be necessary to ensure that the provisions of this Law are complied with; [g] issue an improvement notice requiring the owner or operator of any manufacturing plant, undertaking or establishment which pollutes or is likely to pollute the environment to take appropriate remedial measures including the installation of new plant or machinery, where necessary; [h] close any manufacturing plant, establishment or other activity which pollutes or is likely to pollute the environment contrary to the provisions of this Law; [i] require the owner or operator of the manufacturing plant, establishment or other activity referred to in paragraph (h) to implement remedial measures that the environmental inspector may, in the notice closing down that manufacturing plant undertaking or establishment, direct; [j] issue an improvement notice requiring the operator of a manufacturing plant, undertaking or establishment to cease activities deleterious to the environment; [k] seize any plant, equipment, vessel, vehicle, manufacturing plant, substance or any other thing which he believes has been used in the commission of an offence against this Law or the regulations made under this Law; or [l] cause a Police officer to arrest a person whom he reasonably believes has committed an offence under this Law. [2] The Agency may, for purposes of enabling environmental inspectors to monitor compliance with the provisions of this Law or the regulations made under this Law, install equipment on any land, premises, manufacturing plant, vehicle or vessel; [3] Any person who tampers with the equipment installed by the environmental inspector, or otherwise interferes with the inspector's duties under this section commits an offence and is liable on conviction of a fine not exceeding 5,000 US DOLLARS or imprisonment for a period not exceeding one year or to both. [4] The environmental inspector shall, in exercising his powers under this Law or the regulations made under this Law, suitably identify himself to a person who owns or operates a manufacturing plant, undertaking or establishment operating contrary to the provisions of this Law.
```

Comment:
Please replace:

* `paragraph (h)` with `paragraph [h]`
* `manufacturing plant undertaking or establishment` with `manufacturing plant, undertaking or establishment`
* `vessel; [3]` with `vessel. [3]`
* `liable on conviction of a fine` with `liable on conviction to a fine`